packages/protocol/contracts/4844/BlobHashReader.yulp

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+//  _____     _ _         _         _
+// |_   _|_ _(_) |_____  | |   __ _| |__ ___
+//   | |/ _` | | / / _ \ | |__/ _` | '_ (_-<
+//   |_|\__,_|_|_\_\___/ |____\__,_|_.__/__/
+
+// An implemenatation of IBlobHashReader
+object "BlobHashReader" {
```

Comment:
@davidtaikocha @Brechtpd take a look a this file and the way it's used. I'm not entire sure whether this works as I still dont' know how to fire a blob tx.